xbout/tests/test_against_collect.py

```diff
@@ -1,14 +1,16 @@
 import pytest
-
 import numpy.testing as npt
 
-from xbout.load import open_boutdataset
+from xbout import open_boutdataset
+from xbout.load import collect as new_collect
```

Comment:
It's probably worth adding `collect` to `xbout/__init__.py` so it can be imported as `from xbout import collect`.

Reply:
Will add now